src/routes/user/index.ts

```diff
@@ -67,6 +71,7 @@ router.post(
 router.post(
   '/user/password',
   bodyValidator(userPasswordSchema),
+  // authenticationGate(true, false, (req) => req.body.ticket !== undefined), // this is done in the handler because the handler has an auhtenticated and unauthenticated mode.............
```

Comment:
actually, I think I want it there because I spent 15 minutes trying to figure why we weren't using the middleware there (see the comment added in the end of the line)